inc/commondbchild.class.php

```diff
@@ -881,4 +881,25 @@ public static final function getItemField($itemtype): string {
 
       throw new \RuntimeException('Cannot guess field for itemtype ' . $itemtype . ' on ' . static::class);
    }
+
+   protected function autoinventoryInformation() {
+      echo "<td>".__('Automatic inventory')."</td>";
+      echo "<td>";
+      if ($this->fields['id'] && $this->fields['is_dynamic']) {
+         if (Plugin::haveImport()) {
```

Comment:
As a plugin can have import for some items and will certainly have no `autoinventory_information` on many items, this check is not really relevant.
Thus, as we handle emptyness of `autoinventory_information` hook result, we do not need this condition.
```suggestion
```